editor/src/components/canvas/canvas-strategies/strategies/grid-helpers.ts

```diff
@@ -174,6 +178,45 @@ export function runGridRearrangeMove(
   }
 }
 
+export function gridPositionToValue(p: GridPosition | null | undefined): string | number | null {
+  if (p == null) {
+    return null
+  }
+  if (p === 'auto') {
+    return 'auto'
+  }
+
+  return p.numericalPosition
+}
+
+export function setGridProps(
```

Comment:
(edit: I see you just moved this function, but)

I think the function name should reflect that this creates commands, for readability inside the strategy (when debugging a strategy, i often look for "where are the commands created?")

Reply:
I changed the name to `setGridPropsCommands` in https://github.com/concrete-utopia/utopia/pull/6038/commits/c22a06a5254ddb695189e3eb48342f73148ef37e